gensim/models/tfidfmodel.py

```diff
@@ -433,6 +433,10 @@ def load(cls, *args, **kwargs):
             model.slope = 0.65
             logger.info('older version of %s loaded without slope arg', cls.__name__)
             logger.info('Setting slope to %s.', model.slope)
+        if not hasattr(model, 'smartirs'):
+            model.smartirs = None
+            logger.info('older version of %s loaded without smartirs arg', cls.__name__)
+            logger.info('Setting smartirs to %s.', model.smartirs)
```

Comment:
Not much point in this second logging statement, IMO.

Reply:
@mpenkov  The second logging statement is following the style of the other backwards compatibility code.